packages/fury-adapter-oas3-parser/test/unit/parser/oas/parseServersArray-test.js

```diff
@@ -15,21 +15,23 @@ describe('#parseServersArray', () => {
   it('warns when it is not an array', () => {
     const servers = new namespace.elements.Object();
 
-    const parseResult = parse(context, servers);
+    const name = 'parent-name';
```

Comment:
You could perhaps define this at the top of the file as it's repeated a few times.

Reply:
moved to top